drivers/hmis/app/graphql/mutations/update_external_form_submission.rb

```diff
@@ -8,24 +8,71 @@ module Mutations
   class UpdateExternalFormSubmission < CleanBaseMutation
     argument :input, Types::HmisSchema::ExternalFormSubmissionInput, required: true
     argument :id, ID, required: true
+    argument :project_id, ID, required: false # not required for backwards compatibility
 
-    field :external_form_submission, Types::HmisSchema::ExternalFormSubmission, null: false
+    field :external_form_submission, Types::HmisSchema::ExternalFormSubmission, null: true
```

Comment:
Changed to nullable because null may be returned if there are errors. I think this change could be problematic since it's not backwards compatible from the frontend's perspective, but maybe it's ok if the external form feature is not actively in use yet?